src/js/hooks/list-pages/product/useProductsListTableData.jsx

```diff
@@ -0,0 +1,79 @@
+import fileDownload from 'js-file-download';
+import _ from 'lodash';
+import queryString from 'query-string';
+
+import useTableData from 'hooks/list-pages/useTableData';
+import apiClient from 'utils/apiClient';
+
+const useProductsListTableData = (filterParams) => {
+  const url = '/openboxes/api/products';
+  const messageId = 'react.productsList.fetch.fail.label';
+  const defaultMessage = 'Unable to fetch products';
+  const getSortingParams = tableState => (tableState.sorted.length > 0 ?
```

Comment:
I also feel like `getSortingParams` can be moved to **useTableData**.
The logic for this seems very generic across all of the _use*TableData_ hooks and you would only need to pass the `sortColumn: 'lasUpdated'` and  `sortOrder: 'desc'` as params